docs/samples/python/customizing_presidio_analyzer.ipynb

```diff
@@ -462,11 +462,11 @@
     ")\n",
     "\n",
     "# Analyze in different languages\n",
-    "results_spanish = analyzer.analyze(text=\"Mi nombre es David\", language=\"es\")\n",
+    "results_spanish = analyzer.analyze(text=\"Mi nombre es Morris\", language=\"es\")\n",
     "print(\"Results from Spanish request:\")\n",
     "print(results_spanish)\n",
     "\n",
-    "results_english = analyzer.analyze(text=\"My name is David\", language=\"en\")\n",
+    "results_english = analyzer.analyze(text=\"My name is Morris\", language=\"en\")\n",
```

Comment:
are you a space cowboy? :)

Reply:
lol why?

Reply:
:) https://www.youtube.com/watch?v=h66V1hYAjG0